Makefile

```diff
@@ -33,6 +33,10 @@ BUILD_DIR ?= .
 # Release information.
 GPG_KEYID ?=
 
+# ROOT / ROOTLESS tests. Turn them off by setting the env vars to '0'.
+ROOT ?= 1
+ROOTLESS ?= 1
```

Comment:
I don't really like this to be honest. Is there a clear benefit to doing this over just having a separate target for `test-integration-root` and `test-integration-rootless`?